lib/membrane_rtsp/rtsp.ex

```diff
@@ -1,50 +1,141 @@
 defmodule Membrane.RTSP do
-  @moduledoc "RTSP Session"
+  @moduledoc """
+  Functions for interfacing with a RTSP session
+  """
   use GenServer
 
-  import Membrane.RTSP.Logic
-
   alias Membrane.RTSP
-  alias Membrane.RTSP.Logic.State
   alias Membrane.RTSP.{Request, Response, TCPSocket}
 
   @type t() :: pid()
 
+  @type options() :: [option()]
+  @type option() ::
+          {:connection_timeout, non_neg_integer()} | {:response_timeout, non_neg_integer()}
+
   @default_rtsp_port 554
+  @user_agent "MembraneRTSP/#{Mix.Project.config()[:version]} (Membrane Framework RTSP Client)"
+
+  defmodule State do
+    @moduledoc "Struct representing the state of RTSP session"
```

Comment:
Should be `@moduledoc false`. Generate the docs and make sure that all and only public API is there. I think we can also remove `Membrane.RTSP.Server.Logic` and `Membrane.RTSP.Server.Logic.State` from there